vllm/v1/core/sched/scheduler.py

```diff
@@ -62,13 +60,31 @@ def __init__(
             self.scheduler_config.max_num_batched_tokens
         self.max_model_len = self.scheduler_config.max_model_len
 
+        # create connector
```

Comment:
nit: Can you please add a brief comment on what the KV cache connect is?
```suggestion
        # Create KV cache connector if necessary. The KV cache connector
        # is used to transfer KV cache to different types of storages.
```

Reply:
Np